Makefile.ide

```diff
@@ -159,6 +159,18 @@ $(IDETOPBYTE): ide/idetop.ml $(LINKCMO) $(LIBCOQRUN) $(IDETOPCMA)
 			 $(SYSMOD) -package camlp5.gramlib \
 			 $(LINKCMO) $(IDETOPCMA) $(BYTEFLAGS) $< -o $@
 
+ifeq ($(ARCH),Darwin)
+KEYSRC:=ide/config_files/MacOS/default_accel_map
```

Comment:
You could maybe do `KEYSRC:=ide/config_files/$(ARCH)/default_accel_map` , but I am not expert.